users/validators.py

```diff
@@ -23,3 +23,21 @@ def __call__(self, users_birth_date: Optional[datetime.date]) -> Optional[dateti
             if not self.MINIMAL_ACCETABLE_AGLE <= age <= self.MAXIMAL_ACCEPTABLE_AGE:
                 raise ValidationError(ValidationErrorText.VALIDATION_ERROR_AGE_NOT_ACCEPTED)
         return users_birth_date
+
+
+@deconstructible
+class UserDomainEmailValidation:
+    def __init__(self, domain_list: list) -> None:
+        self.domain_list = domain_list
```

Comment:
Personally I would prefer to delete from here `__init__` method and get acceptable domain_list from settings. I am not sure that this is possible, but I think it is worth to try.

Reply:
It can also work both ways - if not list is provided, get one from settings.